main.css

```diff
@@ -14,3 +14,4 @@
 @import '/css/pages/auth.css';
 @import '/css/pages/landingPage.css';
 @import '/css/pages/cart-page.css';
```

Comment:
suggestion - maintain a consistent file naming. In landing page you have used camelcase & in cart page you have used hyphen. Anything works fine but I will suggest using a single method for file naming.

Reply:
sure will make the changes.